### ISSUE TITLE
Problema amb detalls de la comanda

### ISSUE THREAD

**Issue by mamuts** [jmueller17/Aixada#270]
Bones,

Al actualitzar la web ens hem trobat que quan cliques a "Gestiona"->"Comandes" i tries qualsevol de les comandes servides abans et mostrava el que s'havia servit amb les modificacions de cada producte i el que s'havia demanat en la comanda original, actualment ens apareix totes les dades de la comanda original a 0. Fins hi tot s'han esborrat totes les dades de comandes anteriors.

He mirat a la base de dades però no he sapigut trobar el que. Si sabeu com solventar-ho ens farieu un favor ;)

Salut!

**Comment by jorix** [comment]
> Al actualitzar la web...

Concretament a què et refereixes?, de servidor?, de versió?  
Indiqueu pasos que heu fet i versions que useu...

Heu mirat en el navegador si falla alguna petició de xarxa (F12)?

**Comment by mamuts** [comment]
A veure si ho faig bé...

Hem actualitzat la web que tením amb la branca master actual per evitar els problemes amb el php 7.2. Vam actualitzar la base de dades amb l'arxiu dbUpgradeTo2.8.sql i també vam executar a través de phpmyadmin l'arxiu aixada_queries_all.sql.

Ara, com he comentat, un dels problemes que ens trobem és que no ens apareix al revisar la comanda les dades originals de la comanda. Sinó que ens apareix tot en (0). Com per exemple:

[www.olidemosketa.org/CapturaWeb.png](url)

Gràcies per l'ajuda!

**Comment by jorix** [comment]
> ... Sinó que ens apareix tot en (0)  
  
Ups!, crec que això és normal a la 2.8, jo sempre ho he vist així (imagino que està relacionat amb que *la taula `aixada_order_to_shop` ja no és temporal a la `2.8` i a la `master`*),    
@jmueller17 és així?  
(podria ser convenient que algú revises el codi per no confondre...)  

@mamuts teníeu una versió prèvia a la 2.8, oi?  
@mamuts Proveu **imprimir a comandes**: Al imprimir indica la quantitat rebuda i si s'havia demanat diferent **SÍ** que ho mostra una casella prèvia en gris, és el que esperàveu? 

> ...un dels problemes que ens trobem...

Teniu altres problemes? Quins?

**Comment by mamuts** [comment]
@jorix Sí, teníem una versió prèvia a la 2.8. He provat el que em comentes de imprimir les comandes i si que surt correctament però no hi hauria possibilitat de fer que es mostrés com abans al clicar sobre la comanda?

i sí, hem trobat algún que altre problema, entre ells el fet que la compte Cashbox no resta les validacions que es fan, fent que cada cop tingui un valor més gran perquè només suma els ingressos.

Per la resta crec que de moment anem fent. Si trobem més coses ja les comentarem! ;)

Moltes gràcies per l'ajuda @jorix

**Comment by jorix** [comment]
> però no hi hauria possibilitat de fer que es mostrés com abans al clicar sobre la comanda?

Ups!, jo personalment no tinc previst dedicar ara temps a desenvolupament de Aixada.  
Sí que estic implicat en donar suport.  
Actualment considero que Aixada és raonablement estable. 

> la compte Cashbox no resta les validacions que es fan..

No!, jo no recordo que ho fes, no sé si abans... (@jmueller17 te suena esto antes lo hiciera?)

Ara la plana de gestionar diners ha canviat completament, permet més coses que abans. El treball amb els diners està gestionat per operacions i cada operació té un o dos comptes afectats.

Operacions de diner:

Operació| Codi| *Saldo* Caixa Bancs| *Saldo* UF| *Saldo* Proveidors|  Tipus moviment
---------| ---------| ---------| ---------| ---------| ---------
**Ingressos i UFs**| - - -| - - -| - - -| - - -| - - -
Ingrés altres| `deposit_others`| +1| | | deposit
Ingrés de UF| `deposit_uf`| +1| +1| | deposit
Devolució a UF| `refund_uf`| -1| -1| | withdrawal
Càrrec a UF `validate`| `debit_uf`| | -1| | bill
Anul·lar Càrrec a UF| `a_debit_uf`| | +1| | bill
**Pagaments**| - - -| - - -| - - -| - - -| - - -
Pagament altres| `withdraw_others`| -1| | | withdrawal
**Proveïdors (opcional)**| `$accounts=array( 'use_providers'=> true );`| - - -| - - -| - - -| - - -
*Pagament a Prv.*| *`pay_pr`*| -1| | -1| withdrawal
*Anul·lar Pag.Prv.*| *`a_pay_pr`*| +1| | +1| withdrawal
*Factura de Prv. `distribute_and_validate`*| *`invoice_pr`*| | | +1| bill
*Anul·lar Factura Prv.*| *`a_invoice_pr`*| | | -1| bill
**Caixa i Bancs**| - - -| - - -| - - -| - - -| - - -
Moure| `move`|  +1  & -1| | | cash
Ajustar saldo| `correction`|  import inicial | | | correction
